layouts/partials/head.html

```diff
@@ -19,11 +19,12 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 {{ hugo.Generator }}
-{{ if eq (getenv "HUGO_ENV") "production" }}
+{{- $outputFormat := partial "outputformat.html" . -}}
```

Comment:
These changes mirror the theme's file, `layout/partials/head.html`.
Is there a missing case or condition that needs to be checked?